src/CodeCoverage.php

```diff
@@ -170,8 +162,12 @@ public function filter(): Filter
      */
     public function getData(bool $raw = false): ProcessedCodeCoverageData
     {
-        if (!$raw && $this->includeUncoveredFiles) {
-            $this->addUncoveredFilesFromFilter();
+        if (!$raw) {
+            if ($this->processUncoveredFiles) {
+                $this->processUncoveredFilesFromFilter();
```

Comment:
I've run the tests locally, and as codecov tells, there's no test that covers this line nor the `processUncoveredFilesFromFilter` method